index.html

```diff
@@ -950,9 +950,40 @@ <h4>
     </pre>
   </section>
 
+  <section><h3>General information about writing data</h3>
+    <p>
+      Writing data is generally straight-forward, but there are a few
+      gotcha's around how writing works with NFC.
+    </p>
+    <p>
+      An NFC reader works by polling, so in order to be able to
+      write to a tag, a tag first needs to be found and read, which
+      means that polling needs to be initialized first.
+    </p>
+    <p>
+      If polling is not initiated already by first calling `scan()`,
+      then the `write()` method will initiate it temporarily until a
+      tag was found and read, and writing to it was attempted.
+    </p>
```

Comment:
This is not explicitly said in the write() steps, but it's indeed implicitly assumed.

Reply:
It's not said in write steps either which might be the reason why we have the read after writing if scan is not called

Reply:
Also we should probably talk about the polling and NFC Read operation as used in the NFC specs (at least tnep)